### PR TITLE
settings rename / cleanup

### PR DESCRIPTION
**Pull request by snopoke**
http://manage.dimagi.com/default.asp?186041
Rename / cleanup settings to ensure obfuscation by celery API: https://github.com/celery/celery/blob/master/celery/app/utils.py

@czue 
cc @sravfeyn

Corresponding ansible change: https://github.com/dimagi/commcarehq-ansible/pull/249

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -594,17 +591,6 @@
...
-
-# unicel sms config
-UNICEL_CONFIG = {"username": "Dimagi",
-                 "password": "changeme",
-                 "sender": "Promo"}
-
-# mach sms config
-MACH_CONFIG = {"username": "Dimagi",
-               "password": "changeme",
-               "service_profile": "changeme"}
```
Comment (snopoke) [inline settings.py]: @gcapalbo can you confirm that these sms settings are no longer in use

**Comment by czue** [comment]
```python
HIDDEN_SETTINGS = re.compile(
    'API|TOKEN|KEY|SECRET|PASS|PROFANITIES_LIST|SIGNATURE|DATABASE',
    re.IGNORECASE,
)
```

that is so terrible.

**Comment by snopoke** [comment]
Yea, I though about doing a PR to celery to allow specifying a custom cleaning fn. May still do so.

File: docker/localsettings.py [hunk 2]
```diff
@@ -125,9 +113,9 @@
 
 # Set your analytics IDs here for GA and pingdom RUM
 ANALYTICS_IDS = {
-    'GOOGLE_ANALYTICS_ID': '*******',
-    'PINGDOM_ID': '*****',
-    'ANALYTICS_ID_PUBLIC_COMMCARE': '*****',
+    'GOOGLE_ANALYTICS_API_ID': '*******',
+    'PINGDOM_API_ID': '*****',
+    'ANALYTICS_API_ID_PUBLIC_COMMCARE': '*****',
```
Comment (benrudolph) [inline docker/localsettings.py]: don't even think we use this anymore, but might be good to keep around?

**Comment by benrudolph** [comment]
yikes!

File: settings.py (hunk 1 above)
Comment (gcapalbo) [inline settings.py]: yup, UNICEL_CONFIG and MACH_CONFIG aren't used from settings anymore

Comment (gcapalbo) [inline settings.py]: (and same with SMS_GATEWAY_URL and SMS_GATEWAY_PARAMS)

**Comment by sravfeyn** [comment]
Looks good.

**Comment by snopoke** [comment]
:white_check_mark: backwards compatible changes made to localsettings on prod and india

**Pull request merged**